packages/@remax/core/src/index.js

```diff
@@ -28,9 +33,28 @@ function setData(rootContext) {
 
   const pureObject = clone(rootContext[REMAX_ROOT_BACKUP]);
 
-  rootContext.setData({
-    [REMAX_ROOT]: pureObject,
-  });
+
+  if (lastData) {
+    // 更新了 lastData 等待 setData 发生即可
+    lastData = pureObject;
```

Comment:
this will be executed many times, 7,12, or 24, Could you explain me why? Thank you.